internal/helm/helm.go

```diff
@@ -122,47 +124,55 @@ func makeStack(appname string, targetDir string, data []byte, stackVersion strin
 	if err := os.MkdirAll(filepath.Join(targetDir, "templates"), 0755); err != nil {
 		return err
 	}
-	var stack interface{}
+	var stackData []byte
 	switch stackVersion {
 	case V1Beta2:
 		stackSpec := templateconversion.FromComposeConfig(rendered)
-		stack = templatev1beta2.Stack{
-			TypeMeta:   typeMeta(stackVersion),
+		stack := templatev1beta2.Stack{
+			TypeMeta: templatev1beta2.TypeMeta{
+				Kind:       "Stack",
+				APIVersion: "compose.docker.com/" + stackVersion,
+			},
 			ObjectMeta: objectMeta(appname),
 			Spec:       stackSpec,
 		}
+		templatetypes.ProcessTemplate = toGoTemplate
+		stackData, err = yaml.Marshal(stack)
+		if err != nil {
+			return err
+		}
 	case V1Beta1:
+		templatetypes.ProcessTemplate = toGoTemplate
 		composeFile, err := yaml.Marshal(rendered)
 		if err != nil {
 			return err
 		}
-		stack = v1beta1.Stack{
+		stack := v1beta1.Stack{
 			TypeMeta:   typeMeta(stackVersion),
```

Comment:
Why not use the same custom `TypeMeta` used above ? :thinking:

Reply:
(and thus not doing the marshal/unmarshal/marshal flow)

Reply:
because it's a v1beta1 stack type taken from cli, we don't have control over it.

Reply:
ok, why not having this "small" piece of `v1beta1` stack in here, i.e.

```
type v1beta1Stack struct {
    TypeMeta: …
}
```

And use the same type…